Makefile

```diff
@@ -72,6 +72,9 @@ test-python:
 test-python-integration:
 	FEAST_USAGE=False IS_TEST=True python -m pytest -n 8 --integration sdk/python/tests
 
+test-python-universal-contrib:
+	PYTHONPATH='.' FULL_REPO_CONFIGS_MODULE=sdk.python.feast.infra.offline_stores.contrib.contrib_repo_configuration FEAST_USAGE=False IS_TEST=True python -m pytest -n 8 --integration --universal sdk/python/tests
```

Comment:
Why is `PYTHONPATH` added here?

Reply:
It just temporarily changes the path so that we can use the module from the feast directory. I was having trouble importing the repo configs module as an absolute path using the root directory.